app/views/flexible_criteria/index.html.erb

```diff
@@ -9,16 +9,17 @@
     <%= t('criteria.marking_scheme', identifier: @assignment.short_identifier) %>
   </h1>
   <div class='heading_buttons'>
-    <%= link_to t('add_criterion'), 
-          'javascript:void(0);',
-          onclick: "if(document.getElementById('new_criterion') != null) {
-          document.getElementById('new_criterion_prompt').focus();
-        } else {
-          jQuery.ajax({
-            url: '" +new_assignment_flexible_criterion_path(@assignment.id) + "',
-            type: 'GET'
-          });
-        }"%>
+    <%= link_to t('criteria.add_criterion'),
+      'javascript:void(0);',
+      onclick: "if(document.getElementById('new_criterion') != null) {
```

Comment:
How are we going to handle the creation of either a flexible criterion or a rubric criterion for the same assignment? Are we going to create two different links that the user can choose from?